packages/styles/scss/patterns/sub-patterns/content-section/_content-section.scss

```diff
@@ -0,0 +1,74 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '../../../globals/imports';
+
+@mixin themed-items {
+  color: $text-01;
+  background: $ui-background;
+}
+
+@mixin content-section {
+  .#{$prefix}--content-section {
+    &__grid {
+      @include carbon--make-container;
+    }
+    &__row {
+      @include carbon--make-row;
+    }
+    &__left,
+    &__children {
+      @include carbon--make-col-ready;
+    }
+    &__left {
+      @include carbon--breakpoint(lg) {
+        @include carbon--make-col(4, 16);
+      }
+    }
+    &__heading {
+      margin-top: $carbon--layout-01;
+      margin-bottom: $carbon--layout-05;
+
+      @include type-style('expressive-heading-02');
+      @include carbon--breakpoint('lg') {
+        margin-top: $carbon--layout-02;
+      }
+    }
+    &__children {
+      @include carbon--breakpoint(lg) {
+        @include carbon--make-col(12, 16);
+      }
+    }
+
+    @include carbon--breakpoint('md') {
+      padding-bottom: $carbon--layout-06;
+    }
+
```

Comment:
Where did the `lg` `padding-top: $carbon--layout-04` go?

Reply:
removed.

Reply:
Why? It's part of the design spec. I think you might have removed the wrong one. The one above was supposed to be removed.

Reply:
updated.